src/components/Pickers/BasePicker.Props.ts

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { IPickerItemProps } from './PickerItem.Props';
+
+export interface IBasePickerProps<T> extends React.Props<any> {
+  /**
+   * Function that specifies how the selected item will appear.
+   */
+  onRenderItem?: (props: IPickerItemProps<T>) => JSX.Element;
+  /**
+   * Function that specifies how the suggestion will appear
+   */
+  onRenderSuggestion?: (props: T) => JSX.Element;
+  /**
+   * A callback for what should happen when a person types text into the input.
+   * Returns the already selected items so the resolver can filter them out.
+   */
+  onResolveSuggestions: (filter: string, selectedItems?: T[]) => any[];
+  /**
+   * Initial items that have already been selected and should appear in the people picker.
+   */
+  startingItems?: T[];
+  /**
+   * A callback for when the selected list of items changes.
+   */
+  onChange?: (items: T[]) => void;
+  /**
+   * A callback to get text from an item. Used to autofill text in the pickers.
+   */
+  getTextFromItem?: (item: T) => string;
+  /**
+   * A callback that get's the rest of the results when a user clicks get more results.
```

Comment:
Typo in comments. Also I think this should be part of the suggestions props right? As in, the caller wouldn't need to pass this into the picker, they would pass into suggestions maybe? Not sure.